src/components/User.vue

```diff
@@ -1,38 +1,144 @@
-<script setup lang="ts">
-  import { createApp } from 'vue';
+<script lang="ts">
+import { defineComponent, ref, watchEffect } from 'vue';
+import Search from './Search.vue';
 
+interface UserInfo {
+  avatar_url: string;
+  name: string | null;
+  login: string;
+  created_at: string;
+  bio: string | null;
+  public_repos: number;
+  followers: number;
+  following: number;
+  location: string | null;
+  blog: string | null;
+  twitter_username: string | null;
+  company: string | null;
+}
+
+export default defineComponent({
```

Comment:
It would also be nice, if search is empty, to not see error, that user is not found. It should be some default view.